mongorestore/main/mongorestore.go

```diff
@@ -8,47 +8,23 @@
 package main
 
 import (
-	"fmt"
-	"os"
-	"time"
-
-	"github.com/mongodb/mongo-tools/common/db"
-	"github.com/mongodb/mongo-tools/common/log"
-	"github.com/mongodb/mongo-tools/common/options"
-	"github.com/mongodb/mongo-tools/common/progress"
-	"github.com/mongodb/mongo-tools/common/signals"
-	"github.com/mongodb/mongo-tools/common/util"
+	"github.com/mongodb/mongo-tools-common/log"
+	"github.com/mongodb/mongo-tools-common/signals"
+	"github.com/mongodb/mongo-tools-common/util"
 	"github.com/mongodb/mongo-tools/mongorestore"
-)
 
-const (
-	progressBarLength   = 24
-	progressBarWaitTime = time.Second * 3
+	"os"
 )
 
 func main() {
-	// initialize command-line opts
```

Comment:
most of the options parsing was moved to `ParseArgs` in `options.go` and other logic to `New` in `mongorestore.go`.

Reply:
Thanks.  That will help make [TOOLS-2195](https://jira.mongodb.org/browse/TOOLS-2195) easier, as well.